python/tests/unit/arcticdb/version_store/test_deletion.py

```diff
@@ -285,3 +285,28 @@ def test_delete_date_range_pickled_symbol(lmdb_version_store):
     assert lmdb_version_store.is_symbol_pickled(symbol)
     with pytest.raises(InternalException) as e_info:
         lmdb_version_store.delete(symbol, (idx[1], idx[2]))
+
+
+def test_write_batch_prune_previous_versions(lmdb_version_store):
```

Comment:
Surely this isn't a unit test?!

Reply:
Moved to test_basic_version_store.py, where other tests verify the same option for non-batch methods